client/blockchain.go

```diff
@@ -188,28 +188,30 @@ func (n *NetworkRegistry) GetNetworks(env *environment.Environment) (*Networks,
 }
 
 // NewChainlinkClients creates new chainlink clients
-func NewChainlinkClients(e *environment.Environment) ([]Chainlink, error) {
+func NewChainlinkClients(e *environment.Environment, charts []string) ([]Chainlink, error) {
```

Comment:
To avoid changing all the tests to always need to pass in a string slice of what 9/10 times will be the same, create two functions, eg:

```go
func NewChainlinkClients(e *environment.Environment) ([]Chainlink, error) {
  return NewChainlinkClientsByCharts(e, []string{"chainlink"})
}
```